frontend/rust-lib/flowy-database2/src/services/database/database_editor.rs

```diff
@@ -944,17 +944,25 @@ impl DatabaseEditor {
             .last()
             .map(|row_detail| row_detail.row.id.clone())
         };
+        if let Some(row_id) = to_row.clone() {
+          self.move_row(view_id, from_row.clone(), row_id).await;
+        }
 
-        if let Some(row_id) = to_row {
-          self.move_row(view_id, from_row, row_id).await;
+        if from_group.is_some() && from_group.clone().unwrap() == to_group.to_string() {
```

Comment:
from_group.map(|group| group == *to_group).unwrap_or(false)

Reply:
Passing from group id so there is no need for this anymore.